src/test/javascript/spec/service/faq.service.spec.ts

```
@@ -0,0 +1,209 @@
+    afterEach(() => {
+        httpMock.verify();
+    });
+
+    describe('Service methods', () => {
+        it('should create a faq', () => {
+            const returnedFromService = { ...elemDefault };
+            const expected = { ...returnedFromService };
+            service
+                .create(courseId, elemDefault)
+                .pipe(take(1))
+                .subscribe((resp) => (expectedResult = resp));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)